lib/tasks/solid_errors_tasks.rake

```diff
@@ -0,0 +1,18 @@
+namespace :solid_errors do
+  desc "Copy and run migrations from Solid Errors"
+  task install_migrations: :environment do
+    source = File.expand_path("../../solid_errors/db/migrate", __FILE__)
+    destination = File.join(Rails.root, "db", "migrate")
+
+    Dir.glob("#{source}/*.rb").each do |migration|
+      original_filename = File.basename(migration)
+      timestamp = Time.now.strftime('%Y%m%d%H%M%S_')
+      filename_with_timestamp = "#{timestamp}#{original_filename}"
+
+      if Dir.glob("#{destination}/**/*#{original_filename}*").empty?
```

Comment:
Shouldn't need the `/**/` nested directory glob, should we? Does Rails allow for nested directories in the migrations dir?

Reply:
I don't think we do need `/**/`. Removed it in most recent commit